Source/Math/ConvolveGeometry.h

```diff
@@ -461,8 +470,7 @@ class ConvolveGeometry final
             if (mapCount.size() > 1)
                 dimOut *= mapCount[i];
             else if (i == inputShape.GetRank() - 1)
-                dimOut *= mapCount[0];
-
+                dimOut = dimOutNoDilation*mapCount[0];
```

Comment:
Are you sure this part is correct? If there is no dilation, should dimOut *=mapCount[0]?